overlord/snapstate/check_snap.go

```diff
@@ -36,6 +36,8 @@ import (
 var featureSet = map[string]bool{
 	// Support for common data directory across revisions of a snap.
 	"common-data-dir": true,
+	// Support for the "Environment:" feature in snap.yaml
+	"snap-environment": true,
```

Comment:
Can we please call this "snap-env". It's handier to use in a  flag list next to a few other names.